ci/scripts/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-vault
+  make build
```

Comment:
Do we need to have a similar script to dp-deployer
`#!/bin/bash -eux

pushd dp-deployer
  make build 
  cp build/$(go env GOOS)-$(go env GOARCH)/* Dockerfile.concourse ../build
popd`